shared/fs/footer/downloads-container.tsx

```diff
@@ -1,8 +1,10 @@
 import {namedConnect} from '../../util/container'
 import * as FsGen from '../../actions/fs-gen'
+import * as Types from '../../constants/types/fs'
 import Downloads, {DownloadsProps} from './downloads'
 import {isMobile} from '../../constants/platform'
 import {downloadFolder} from '../../util/file'
+import {types} from '@babel/core'
```

Comment:
Is this used?

Reply:
good catch! not sure how it got in there or why the linter didn't catch it